packages/use-intl/src/core/createFormatter.tsx

```diff
@@ -2,47 +2,64 @@ import DateTimeFormatOptions from './DateTimeFormatOptions';
 import Formats from './Formats';
 import IntlError, {IntlErrorCode} from './IntlError';
 import NumberFormatOptions from './NumberFormatOptions';
+import RelativeTimeFormatOptions from './RelativeTimeFormatOptions';
 import TimeZone from './TimeZone';
 import {defaultOnError} from './defaults';
 
-const MINUTE = 60;
+const SECOND = 1;
+const MINUTE = SECOND * 60;
 const HOUR = MINUTE * 60;
 const DAY = HOUR * 24;
 const WEEK = DAY * 7;
 const MONTH = DAY * (365 / 12); // Approximation
+const QUARTER = MONTH * 3;
 const YEAR = DAY * 365;
 
-function getRelativeTimeFormatConfig(seconds: number) {
+const UNIT_SECONDS = {
+  second: SECOND,
+  seconds: SECOND,
+  minute: MINUTE,
+  minutes: MINUTE,
+  hour: HOUR,
+  hours: HOUR,
+  day: DAY,
+  days: DAY,
+  week: WEEK,
+  weeks: WEEK,
+  month: MONTH,
+  months: MONTH,
+  quarter: QUARTER,
+  quarters: QUARTER,
+  year: YEAR,
+  years: YEAR
+} as const;
+
+function resolveRelativeTimeUnit(seconds: number) {
```

Comment:
I didn't add quarter here because I believe we don't want to use the quarter unit in most cases.